11-optimistic-ui/bee-rich/start/package.json

```diff
@@ -63,6 +63,7 @@
     "prisma": "^5.3.1",
     "rimraf": "^5.0.1",
     "tailwindcss": "^3.3.3",
+    "ts-node": "^10.9.1",
```

Comment:
Do we even need to install `ts-node` here if we execute the `ts-node-esm` package with `npx`?